scripts/qa_crowbarsetup.sh

```diff
@@ -4321,12 +4321,41 @@ function onadmin_testpreupgrade
     oncontroller testpreupgrade
 }
 
+function oncontroller_get_fips
+{
+    # checking for cloudver might not get correct results here due to the nature of upgrade
+    local version=`openstack --version 2>&1 | cut -f 2 -d ' '`
+    if [[ "$version" < "3" ]]; then
```

Comment:
just nitpicking: with double brackets you don't need to quote variables:
`[[ $version < "3" ]]`